src/fastga_he/models/weight/cg/cg_variation.py

```diff
@@ -32,6 +35,16 @@ class InFlightCGVariation(om.ExplicitComponent):
     """
 
     def setup(self):
+        if RTA_INSTALLED:
+            empty_weight_variable_name = "data:weight:aircraft_empty:mass"
```

Comment:
And then
```suggestion
            self.empty_weight_variable_name = "data:weight:aircraft_empty:mass"
```
That way you can remove all the other if else statement in the other method of the class